print("error and exception handing")
print("error")